beta-src/src/models/Interfaces.ts

```diff
@@ -0,0 +1,64 @@
+export interface IBoard {
+  territories: ITerritory[];
+  units: IUnit[];
+  myUnits: IUnit[];
+  terrStatus: ITerrStatus[];
+  context: IContext;
+}
+
+export interface ITerritory {
+  id: number;
+  name: string;
+  type: string;
+  supply: string;
+  countryID: number;
+  coast: string;
+  coastParentID: number;
+  smallMapX: number;
+  smallMapY: number;
+  Borders: Array<IBorder>;
```

Comment:
let's use:

```
Borders: IBorder[];
```

unless there is a reason your using Array with a generic type?